Shawn_Baugh_II/flask_stuff/flask_fundamentals/dojo_survey/templates/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <title>Survey</title>
+    </head>
+    <body>
+
+        <form action="/users" method="post">
+            Your Name:
+            <input type="text" name="name" placeholder="Name">
+            Location:
+            <select name="location">
+                <option>LA</option>
+                <option>The Bay</option>
+                <option>Seattle</option>
+                <option>Dallas</option>
+            </select>
+
+            Favorite Language:
+            <input type="text" name="language" placeholder="Fav Language">
+            Comments (please don't):
+            <textarea name="comments" rows="8" cols="80"></textarea>
+            <button type="submit" name="button" value="create_user">Submit</button>
```

Comment:
A submit button does not need a 'name' attribute. Also, you should use `<input type="submit" value="submit"/>` instead (this will give you more default functionality within your html form)